features/related-apps.yml

```
@@ -0,0 +1,8 @@
+name: Related apps
+description: The `related_applications` manifest member lists the OS-specific applications that are related to the current web app, such as a native Android app that provices similar functionality. The `getInstalledRelatedApps()` method lists which of those are installed on the curent device.
```

Suggesting a typo fix here and consistency ("web app manifest member") with other descriptions like this one. I trimmed "native" to stay under the character limit.

```suggestion
description: The `related_applications` web app manifest member lists operating system-specific applications that are related to the current web app, such as an Android app that provides similar functionality. The `getInstalledRelatedApps()` method lists which of those are installed on the current device.
```